server/nodejs/controllers/GeneralService.js

```diff
@@ -1,6 +1,7 @@
 var syscoinClient = require('../index').syscoinClient;
 var varUtils = require('./util/varUtils');
 var commonUtils = require('./util/commonUtils');
+var generalServiceGetInfoUtils = require('./util/generalServiceGetInfoUtils');
 const he = require('he');
```

Comment:
can we use let or const instead of var here  ?

Reply:
agreed - I'm going to revisit that one post-release.  desktop-wrapper seems to complain about certain es6 functionality so I was nervous about putting it here.